doc/tortuga-7-admin-guide.md.raw

```diff
@@ -336,6 +336,10 @@ Commands listed in this "Quickstart" section are intended to be run as the
     instances already have an operating system installed, it is not necessary to
     define the installation media.
 
+    **Note:** Software profiles are created with a default max nodes setting
+    of 25. To change or remove this limit, please read the section on
+    software profiles.
```

Comment:
Can you change this to "use `update-software-profile ... --max-nodes XX`. See section on software profiles for more information" or something to that effect?